src/components/app/footer.tsx

```diff
@@ -37,11 +37,6 @@ const content = [
 ]
 
 const about = [
-  {
-    path: '/pricing',
-    label: 'Pricing',
-    nonMemberRequired: true,
-  },
```

Comment:
Going to add this back. We want this, but we need to make sure it shows up only for users that NEED it.

Reply:
this is not the appropriate solution which is found in how the links are displayed and not that pricing exists at all. The solution can be found closer to the jsx. Please do not remove links to our path to purchase.

Reply:
https://github.com/eggheadio/egghead-next/blob/main/src/components/app/footer.tsx#L106-L131

the solution is in here or the method doing the filtering, not the data